### ISSUE TITLE
As a contributor I would like to include varying electrical prices in place a single value

### ISSUE THREAD

**Issue by BhargavaKrishnaSreepathi** [architecture-building-systems/CityEnergyAnalyst#1679]
In place of a fixed electrical pricing, adding in the varying electrical prices to reflect the Electricity costs. This also needs to have a provision when only data for a single value is available. Thus the config file needs to include a variable to do detailed electricity costs in place of a single one.

**Comment by JIMENOFONSECA** [comment]
@BhargavaKrishnaSreepathi I agree with daren that it could be an schedule. However it depends on how exact you want it to do:
1. it is just ask the user off peak and on-peak tariffs. so just two variables.
2. it consist into creating a schedule (but this has to be done manually every hour of a day).
3. you connect to a certain webservices providing market prices.
4. you define an equation that models those prices.

at the end, the option/model you select can be activated with a Flag in the config file, indicating, whether using fixed, prices or any other more detailed pricing scheeme

**Comment by BhargavaKrishnaSreepathi** [comment]
My current necessity is to use the electricity prices for every given hour as available in Singapore market. So the way I employed is create an excel sheet with the hourly cost details for the entire year and added to our database. I also added a `config flag` for `detailed-electrical-pricing`, which uses the database added. If the flag is false, it creates an array with electricity prices, by repeating the value we have in the LCA database (`numpy.ones(8760) * electricity_price`. 

This setup can also further be extended based on the offpeak and peak tariffs, but currently I am not looking into them. The framework of creating an array either by importing the data or by creating (using the same value), should be good enough for handling offpeak/peak tariff scenario too.
